homeassistant/components/tankerkoenig/coordinator.py

```diff
@@ -31,6 +32,8 @@
 class TankerkoenigDataUpdateCoordinator(DataUpdateCoordinator):
     """Get the latest data from the API."""
 
+    config_entry: ConfigEntry
```

Comment:
You can remove the entry parameter of the coordinator and replace `entry` with `self.config_entry`